zrp/prepare/prepare.py

```diff
@@ -57,7 +57,6 @@ def transform(self, input_data):
             Dataframe to be transformed
```

Comment:
Docstring incomplete. Please add a description of the transform (ie "Transform the data")